#164139674 Configure eslint for Authors Haven to use the Airbnb style guide

chingsley opened the pull request:
#### What does this PR do?
It configures eslint for Author's Haven to follow the Airbnb style guide

#### Description of Task to be completed?
- installs 3 dev-dependencies: eslint, eslint-config-airbnb-base, eslint-plugin-import.
- creates the .eslintrc file to contain the setup code for eslint.

#### How should this be manually tested?
- Run `npm i` to install all the necessary dependencies.
- Create a command callend `lint` in the `script` script  of `package.json`. Example:
    ` "lint" : "eslint <folder-name> --fix" `
- In the command line, while in the root director, run the command `npm run lint`

#### Any background context you want to provide?
- We are using the Airbnb style guide.

#### What are the relevant pivotal tracker stories?
#164139674

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

9jaswag commented on routes/api/users.js:
```diff
@@ -84,4 +84,4 @@ router.post("/users", function(req, res, next) {
 });
 
-module.exports = router;
+module.exports = router;
```
Please fix this EOL issue

chingsley committed:
chore(eslint): configure eslint

- install the required eslint dependencies
- create the .eslintrc file and set up the configuration

 [Finishes #164139674]

9jaswag reviewed (approved):
LGTM!
